sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -149,7 +149,7 @@ export class AssessmentController {
       >
     >,
   ): Promise<MustReturnJobActionAcknowledgement> {
-    await this.studentAssessmentService.updateNOAApprovalStatus(
+    await this.studentAssessmentService.updateAssessmentReady(
```

Comment:
Maybe there was some misunderstanding with some other comment or I am missing something, why was this method renamed?